src/functions.cpp

```diff
@@ -114,6 +115,13 @@ void writeConfig(const std::string& filename)
     boost::property_tree::write_ini(filename, tree);
 }
 
+void touchFile(const std::filesystem::path& path)
+{
+    FILE* f = fopen(path.c_str(), "ab");
+    (void)fflush(f);
+    (void)fclose(f);
```

Comment:
Those calls are legacy. If we use a modern C++ construct this is irrelevant. Take a look here: https://github.com/viniciusferrao/cloysterhpc/blob/9867da562dc80d52809e5ba4da17cf6727aebcd1/src/functions.cpp#L216